### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by PrangonGhose**
### Functionalities Implemented in this pull request
- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,74 +1,69 @@
...
-    description: 'Take the dog to walk',
-    completed: false,
-    index: 5,
-  },
-  {
-    description: 'Pay the bill',
-    completed: false,
-    index: 60,
-  },
-  {
```
Comment (Yazino12) [inline src/index.js:24]: - [ ] Well done removing the hardcoded items from the task array ✅.

**Review by pinkmoon25 — APPROVED** [review]
Hi @PrangonGhose, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linters checks :heavy_check_mark: 
Professional Readme :heavy_check_mark: 
All functionality works :heavy_check_mark: 

## Optional suggestions

[OPTIONAL] Great job implementing the functionalities evrything is working well :clap: , I think it would be better if you either use only the _**delete icon**_ or hide the _**delete icon**_ behind the _**three dots,**_ you can make it like when somebody clicks the _**three vertical dots icon**_ the t_**hree dot icon**_ changes to _**delete or trash can icon**_, using _**display none and display block**_ appropriately, It will improve the UI of the application currently they are both together like this :point_down: 
image

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/todoObject.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// make object instance
+export default class Todo {
+  constructor(description, completed = false, index) {
```
Comment (Yazino12) [inline src/todoObject.js:3]: - [ ] Good job setting the default state of the completed property to `false` ✅

**Review by Yazino12 — APPROVED** [review]
## Hi @PrangonGhose 🖐🏾,

You have done a great job implementing the milestone features 🎯, and there is nothing else left to stay other than it's time to merge it 🚀🚀.

### To Highlight:
- Descriptive pull request title and summary ✔
- Correct use of GitHub flow ✔
- Organized code ✔
- Descriptive Readme ✔

# STATUS: APPROVED ✅✅✅
## Congratulations 🎉🎉🎉 your Todo app works as expected and you went above and beyond the requirements by delivering such a high-quality code, Keep up the good work. 👊🏽.
image
image

_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**